docs/_templates/layout.html

```diff
@@ -28,6 +28,22 @@
     .wy-side-nav-search>div.version {
       color: rgba(0, 0, 0, 0.3)
     }
+
+    .wy-nav-side {
+      left: auto;
+    }
+
+    .wy-grid-for-nav {
+      max-width: 1200px;
+      position: relative;
+      margin: auto;
+    }
+
+    .wy-nav-content {
+      max-width: 100%;
+      margin-left: auto;
+      margin-right: auto;
+    }
```

Comment:
Somehow the sidebar lost the ability to automatically collapse below certain window width. It would only affect users of mobile phones - I can hardly imagine anyone using a browser with <1000px width on any computer or even tablet.
I asked Lynsey to see if we have any info on mobile market share.